src/renderer/services/authService/types.ts

```diff
@@ -0,0 +1,14 @@
+export type AuthUser = {
+  uid: string;
+  displayName: string;
+};
+
+export interface AuthService {
```

Comment:
Need to implement `getIdToken`.

Reply:
I'll add that with an incoming PR for the `SlippiBackendService`.

Reply:
Should be resolved in d94ff43